application/build.gradle.kts

```diff
@@ -6,8 +6,34 @@ buildscript {
         compileSdk = 31,
         dataBinding = true,
         extraPlugins = listOf(
-            "androidx.navigation:navigation-safe-args-gradle-plugin:2.4.2",
-            "com.google.firebase:firebase-crashlytics-gradle:2.5.0",
+            "androidx.navigation:navigation-safe-args-gradle-plugin:2.5.3",
+            "com.google.firebase:firebase-crashlytics-gradle:2.9.4",
+            "tools.forma.demo:dependencies",
         )
     )
 }
+
+
+tasks.register("build") {
```

Comment:
lets add some comments since its not obvious why we need this tasks

Reply:
do we need this tasks?  
it is a lifecycle task. it's just to be able to call root project task in this way: `./gradlew :build` which is the synonym `./gradlew build`